frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Alert Icon on Topology for Workloads in sidebar and Alerts tab on Monitoring page
+	User should be aware of alerts related to workloads and should see alerts in the sidebar
+
+
+Background:
+    Given user logged into the openshift application
+    And user is at the developer perspective
+
+
+@regression, @smoke, @manual
+Scenario: Alerts tab on the Monitoring page
+    Given user is at Topology page
+    When user opens Monitoring page
+    Then user will see the Alerts tab
+
+
+@regression, @smoke, @manual
+Scenario: Show alert icon on Topology workloads and associated alerts in sidebar in both graph and list view
+    Given user is at Topology page
+    And user has a node with some alerts
+    When user clicks on the alert 
+    Then monitoring tab will get opened on the sidebar
+    And user will see the alert details there
+
+
+@regression, @smoke, @manual
+Scenario: Show firing alert icon on Topology workloads and associated alerts in sidebar in both graph and list view
+    Given user is at Topology page
+    And user has a node with some firing alerts
+    When user clicks on the alert 
+    Then monitoring tab will get opened on the sidebar
+    And user will see that Events section is replaced with Alerts section
+    And user will see the alert details there
```

Comment:
workloads having firing alerts will have an Alerts Section under Monitoring Tab in the Topology Sidepanel. The Alerts will be sorted according to their severity. If there are no firing alerts user will not see the Alerts section. The Events section has been removed.